.github/workflows/ci-cd.yml

```diff
@@ -196,12 +251,12 @@ jobs:
         with:
           node-version: 16
           cache: yarn
-      - run: yarn install --frozen-lockfile
       - name: setup ruby
         uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
-      - run: bundle exec rails db:test:prepare assets:precompile
+      - run: bin/webpack
+      - run: bundle exec rails db:test:prepare
```

Comment:
`assets:precompile` alone adds almost 2 minutes to CI time. I have not verified my assumption yet, but I believe test does not need precompile assets. What our test need is the compiled webpack assets, which on its own takes only 15 seconds.